### PR TITLE
Update error message for non existent file

### PR DESCRIPTION
**Pull request by sushrut111**
# TL;DR

The issue reported, that the exception thrown is not very informative is because the original exception context was not included when the original exception is caught and a new exception is thrown. This change fixes that by preserving the original exception test.

## Type
 - [x] Bug Fix
 - [ ] Feature
 - [ ] Plugin

## Are all requirements met?

 - [x] Code completed
 - [ ] Smoke tested
 - [ ] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description
The issue reported, that the exception thrown is not very informative is because the original exception context was not included when the original exception is caught and a new exception is thrown. This change fixes that by preserving the original exception test.

## Tracking Issue
https://github.com/flyteorg/flyte/issues/1468

## Follow-up issue
_NA_

### CODE REVIEW

**Review by wild-endeavor — DISMISSED** [review]

**Comment by wild-endeavor** [comment]
@sushrut111 sorry but could you run `make fmt` please?  Thanks!

**Comment by sushrut111** [comment]
@wild-endeavor done!

**Review by wild-endeavor — APPROVED** [review]
